Lesson-4-Task-2/src/main/java/Lesson4/Task2/Circle.java

```diff
@@ -0,0 +1,20 @@
+package Lesson4.Task2;
+import Lesson4.Shape;
+public class Circle extends Shape {
+    @Override
+    protected double getArea(int a, int b) {
+        return 0;
```

Comment:
To jest przykład tego problemu, o którym wspominałem wcześniej.
Gdy utworzę sobie obiekt klasy Circle i uruchomię tę metodę to dostanę zero. Nie brzmi to jak poprawnie działający kod.